packages/ts-migrate-plugins/src/plugins/react-props.ts

```diff
@@ -28,11 +30,30 @@ const reactPropsPlugin: Plugin<Options> = {
     const updates: SourceTextUpdate[] = [];
     const getPropsTypeName = createPropsTypeNameGetter(sourceFile);
 
+    // Scan for prop type imports
+    const propTypeIdentifiers: PropTypesIdentifierMap = {};
+    for (const node of sourceFile.statements) {
```

Comment:
instead of iterating 2 times thought the `sourceFile.statements`, could we move this to the loop below?